src/app/[locale]/incident/add/components/IncidentQuestionsLocationForm.tsx

```diff
@@ -48,6 +52,17 @@ export const IncidentQuestionsLocationForm = () => {
     appendAdditionalQuestions()
   }, [formStoreState.main_category, formStoreState.sub_category])
 
+  useEffect(() => {
+    if (formStoreState.isBlocking) {
```

Comment:
Hier gaat eigenlijk hetzelfde fout als genoemd in #50. Door de gekozen structuur moet je nu deze logica 5x implementeren. Het zou top zijn als je dit maar 1x hoeft te doen op het niveau van IncidentQuestinosLocationForm.tsx.